### ISSUE TITLE
Score ~90% on an Accessibility Audit

### ISSUE THREAD

**Issue by demaceo** [demaceo/fitlit#20]
- [ ] You must score as close to 100% in an Accessibility Audit as possible. Be prepared to explain any accessibility audits your application is failing.
